packages/keystone/bin/commands/build.js

```diff
@@ -21,21 +21,25 @@ module.exports = {
   exec: async (args, { exeName, _cwd = process.cwd() } = {}) => {
     process.env.NODE_ENV = 'production';
     let entryFile = await getEntryFileFullPath(args, { exeName, _cwd });
-    let { admin, distDir = keystone.DEFAULT_DIST_DIR } = require(entryFile);
+    let { servers, distDir = keystone.DEFAULT_DIST_DIR } = require(entryFile);
 
     if (args['--out']) {
       distDir = args['--out'];
     }
     let resolvedDistDir = path.resolve(_cwd, distDir);
     await fs.remove(resolvedDistDir);
 
-    if (admin) {
-      console.log('Building Admin UI!');
-      await admin.staticBuild({
-        apiPath: '/admin/api',
-        distDir: resolvedDistDir,
-        graphiqlPath: '/admin/graphiql',
-      });
+    if (servers) {
+      await Promise.all(
+        servers.map((server, index) => {
+          return server.build({
+            apiPath: '/admin/api',
```

Comment:
should we use the config value for api and graphiql path?

Reply:
Yes, this is WIP for a reason.

Reply:
just making sure as previous code also used same thing :)